config/clients/python/template/README_license_disclaimer.mustache

```diff
@@ -0,0 +1 @@
+The code in this repo was auto generated by [OpenAPI Generator](https://github.com/OpenAPITools/openapi-generator) from a template based on the [python experimental template](https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources/python-experimental), licensed under the [Apache License 2.0](https://github.com/OpenAPITools/openapi-generator/blob/master/LICENSE).
```

Comment:
This is outdated, we're no longer using experimental

Reply:
Done